src/configuration/routes/https.md

```diff
@@ -135,13 +135,15 @@ tls:
             -----END CERTIFICATE-----
 ```
 
-## Automatic SSL certificate renewal
+### Automated SSL certificate renewal using Cron
 
-It is possible to set a variable from a cron task, which in turn will cause the site to redeploy.  If the Let's Encrypt certificate is due to expire in less than one month it will automatically renew at that time.  That makes it feasible to set up auto-renewal of the Let's Encrypt certificate.  The caveat is that, like any deploy, there is a very brief downtime (a few seconds, usually) so it's best to do during off-hours.
+If the Let's Encrypt certificate is due to expire in less than one month it will automatically renew at that time.  That makes it feasible to set up auto-renewal of the Let's Encrypt certificate.  The caveat is that, like any deploy, there is a very brief downtime (a few seconds, usually) so it's best to do during off-hours.
 
-You will first need to install the CLI in your application container.  See the section on [API tokens](/gettingstarted/cli/api-tokens.md) for instructions on how to do so.
+> **note**
+>
+> Automated SSL certificate renewal using cron requires to [get an API token and install the CLI in your application container](/gettingstarted/cli/api-tokens.md).
```

Comment:
"requires to get" doesn't work, it needs to be "requires you to get" or "requires getting [and installing]"